noobaa/noobaa-core
# NC | NSFS | Health script missing features

### Explain the changes
1. Add documentation output to the health section in docs
2. Add to health test -
- bucket check - iterate all buckets, and check bucket path exists like in namespace monitor
- account check - iterate all accounts, and check account configuration exists, new_buckets_path exists
3. remove unwanted logs from output
4. add check for rsyslog
```{"service_name":"nsfs","status":"NOTOK","memory":"88.6M","error":{"error_code":"RSYSLOG_SERVICE_FAILED","error_message":"RSYSLOG service is not started properly, Please verify the service with status command."},"checks":{"services":[{"name":"nsfs","service_status":"active","pid":"1204"},{"name":"rsyslog","service_status":"inactive","pid":"0"}],"endpoint":{"endpoint_state":{"response":{"response_code":200,"response_message":"Endpoint running successfuly."},"total_fork_count":0,"running_workers":[]}},"invalid_accounts":[{"name":"naveen","storage_path":"/tmp/nsfs_root_invalid/","code":"STORAGE_NOT_EXIST"}],"valid_accounts":[{"name":"naveen","storage_path":"/tmp/nsfs_root"}],"invalid_buckets":[{"name":"bucket1.json","config_path":"/etc/noobaa.conf.d/buckets/bucket1.json","code":"INVALID_CONFIG"},{"name":"bucket3","storage_path":"/tmp/nsfs_root/bucket3","code":"STORAGE_NOT_EXIST"}],"valid_buckets":[{"name":"bucket2","storage_path":"/tmp/nsfs_root/bucket2"}]}}```

### Issues: Fixed #xxx / Gap #xxx
1. https://issues.redhat.com/browse/MCGI-216

### Testing Instructions:
1. run `node usr/local/noobaa-core/src/cmd/health.js` and each health check is correct against the running nsfs process.
